released/packages/rocq-navi/rocq-navi.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Yoshihiro Imai<y.imai@aist.go.jp>"
+
+homepage: "https://github.com/affeldt-aist/coq2html"
+dev-repo: "git+https://github.com/yoshihiro503/coq2html.git"
+bug-reports: "https://github.com/affeldt-aist/coq2html/issues"
+
+license: "GPL-2.0-or-later"
+synopsis: "Extension of coq2html Document Generator"
+
+description: """
+Extension of coq2html Document Generator"""
+
+build: [make]
+install: [make "BINDIR=%{bin}%" "install"]
+depends: [
+  "ocaml" {>= "4.14"}
+]
+
+tags: [
+  "category:Tools/Document Generator"
+  "keyword:document"
+  "keyword:html"
+  "logpath:"
```

Comment:
```suggestion
  "keyword:document"
  "keyword:html"
```
Unfortunately, there is no predefined category for tools, and `logpath` should not have empty content. So I think all that can be left are the keywords.

Reply:
Thank you.